content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -181,7 +180,7 @@ This is in contrast to the [`AND`](../../../reference/api-reference/prisma-clien
 if you pass in an `undefined` value.
 
 > This is because passing an `undefined` value to an `AND` or `NOT` operator is the same
-as passing nothing at all, meaning the `findMany` query in the example will run without any filters and return all the users.
+> as passing nothing at all, meaning the `findMany` query in the example will run without any filters and return all the users.
```

Comment:
Hm, are we using `>` to be rendered as a blockquote here? This feels a bit weird.

Reply:
Prettier adds a block quote symbol before each line ([repl](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAdKA+ABAFQBYCWAzlsVgEZxgCGArkXFgA7VFEFQDmW1UWABrSgATOADMOcYfywA3agBtajGBB59+AQQByAERkQATgO0B5HAaZxD1VcbIw8jItQC2cdK2at2XLFAhHDm5bHgUFABosd15grEdGfgkRAFleAE8ZAEdlQ3TSPgSsOAAPNyYFRgB3AnCsQyEsGscIWhh1fIkFeEMSXmF6uBhaQz5FBXinLHprIgA6dBAIkAgmGAJoImRQakNDCCqABV2ELZRFKup0reXyGzAAayGAZRYwYOQYBrhluFdKYSiYQAGV4nFo1E4cAAYkZXLZ1lxkCA6KoliA8DBXAoAOqEeBEN5wZ6nAjrWRk9LIsBsdEcBiGGCHGyceHIMSKBjLABWRBKACF7k8YM83HBgZJ2ZyfiBeSVnsFKgBFWiBOBShRckAsXrWZHwwwPYQHKDopiGDgwHEEYSOZAADgADMtzRAGDibExkea4AzZOrljk1czVmcUUQALRQOBSKTowxwHIEBPMyFspAczUyhiuAifb7LXycZWq+AarUwajka22vDIABMyy+1FqwQAwhBXOmQL6AKzomY4KtnTNa2TKACSIgQIrAFrWmhEzxg6Uq5bgAF8N0A))

Reply:
Should this be a block quote though or did we want to really have a `>` here in this file?

Reply:
It's  a block quote

<img width="805" alt="image" src="https://user-images.githubusercontent.com/4734297/124918045-01d78e80-e00e-11eb-9cfe-000fd11d5880.png">